bc_obps/handle_exception.py

```diff
@@ -0,0 +1,20 @@
+"""
+Module: handle_exception.py
+Description: This module handles http exceptions.
+"""
+from django.http import Http404
+from django.core.exceptions import ValidationError, ObjectDoesNotExist
+from registration.utils import generate_useful_error
```

Comment:
This fails pre-commit in CI but is okay locally, thoughts?

Reply:
I believe adding a dot before `registration.utils` will fix the issue. Once we tackle ticket 114 or 176 we no longer face these kinds of issues.